gtda/diagrams/distance.py

```diff
@@ -210,11 +210,11 @@ def transform(self, X, y=None):
         X = check_diagrams(X, copy=True)
 
         if np.array_equal(X, self._X):
-            X2 = None
+            X1 = None
         else:
-            X2 = X
+            X1 = X
 
-        Xt = _parallel_pairwise(self._X, X2, self.metric,
+        Xt = _parallel_pairwise(self._X, X1, self.metric,
```

Comment:
I am a bit stuck: I am reluctant to introduce `X1` and `X2` in `transform`, in the same way as it is in `_parallel_pairwise`, since their role is not symmetric. On the other hand `X` and `X1` does not reflect the "symmetry"  of `_parallel_pairwise`.
Do you have any naming suggestions?